### PR TITLE
update: Add steps for custom container API spec

### PR DESCRIPTION
**Pull request by marcvberg**
_Issue #, if available:_

_Description of changes:_
Add some instructions for using your own docker container specification

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by siegerts — APPROVED** [review]

File: src/pages/cli/usage/containers.mdx [hunk 1]
```diff
@@ -291,6 +291,28 @@ database = os.environ['DATABASE_NAME']
 password = os.environ['DB_PASSWORD']
 ```
 
+## Bring Your Own API Specification
+
+When creating an API using containers, you have the option to bring in your own `Dockerfile`(s) and `docker-compose.yml`. When selecting the image you want to use for your REST API, choose `Custom`. Afterward, you can choose when you want to deploy changes to your container.
```
Comment (sachscode) [inline src/pages/cli/usage/containers.mdx:296]:
```suggestion
When creating an API using containers, you have the option to bring in your own `Dockerfile`(s) and `docker-compose.yml`. When selecting the docker image for your REST API, choose `Custom` and then select when to deploy code and configuration changes to your container.
```

File: src/pages/cli/usage/containers.mdx [hunk 2]
```diff
@@ -291,6 +291,28 @@ database = os.environ['DATABASE_NAME']
...
+
+```console
+? What image would you like to use Custom (bring your own Dockerfile or docker-compose.yml)
+? When do you want to build & deploy the Fargate task 
+> On every "amplify push" (Fully managed container source) 
+  On every Github commit (Independently managed container source) 
+  Advanced: Self-managed (Learn more: docs.amplify.aws/cli/usage/containers) 
+```
+
+The CLI will provide you with some next steps to follow, and you can take your existing specification and allow the CLI to bring it into your project by populating the `src` directory with your specification.
```
Comment (sachscode) [inline src/pages/cli/usage/containers.mdx:306]:
```suggestion
The CLI will provide you with the next steps to bring your existing docker specification and source files into your project by populating the `src` directory.
```

File: src/pages/cli/usage/containers.mdx [hunk 3]
```diff
@@ -291,6 +291,28 @@ database = os.environ['DATABASE_NAME']
 password = os.environ['DB_PASSWORD']
 ```
 
+## Bring Your Own API Specification
```
Comment (sachscode) [inline src/pages/cli/usage/containers.mdx:294]:
```suggestion
## Bring Your Own Container Specification For Your REST API
```

**Review by sachscode — COMMENTED** [review]
Mostly LGTM, requested a few changes.

**Review by sachscode — APPROVED** [review]
LGTM

**Pull request merged**